shoes-core/lib/shoes/dsl.rb

```diff
@@ -583,7 +583,7 @@ def keyrelease(&blk)
     end
 
     def append(&blk)
-      blk.call if blk
+      yield if blk
```

Comment:
if we change this we could/should remove the `&blk` and instead check `block_given?` - this avoids the capture of the whole block and therefore also gives a small speed boost :)